src/lib/openapi/spec/event-schema.ts

```diff
@@ -1,51 +1,136 @@
 import { FromSchema } from 'json-schema-to-ts';
 import { tagSchema } from './tag-schema';
+import { IEventTypes } from '../../types';
+import { variantSchema } from './variant-schema';
 
+const eventDataSchema = {
+    type: 'object',
+    additionalProperties: true,
```

Comment:
Do we have some required properties? Maybe
```suggestion
    additionalProperties: true,
    required: ['name', 'type'],
```

Reply:
I don't believe we do. There's nothing preventing you from creating an event of `{}`, if you just want to use the event to trigger some other behaviour.